modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -61,31 +70,162 @@ This can overwrite your customization.
 
+Each of these profiles requires you to set some variables before you use it. Check the [path]``README`` file included with the script to determine which variables you need. For more information about using variables in {ay} scripts, see xref:client-configuration:autoinst-profiles#variables[Variables].
 
+For more information about AutoYast, see https://doc.opensuse.org/projects/autoyast/.
 
-== File Preservation for {kickstart}
 
-If you have many custom configuration files located on a client you want to {kickstart}, you can save them as a list, and associate that list with the {kickstart} profile to be used.
+[[kickstart]]
+== KickStart Profiles
 
-In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > File Preservation] and click btn:[Create File Preservation List].
-Enter a suitable label, and list absolute paths to all files and directories you want to save.
-Click btn:[Create List].
+{kickstart} profiles can be created via a wizard, and offer a larger number of configuration options.
 
-Include the file preservation list in your {kickstart} profile by navigating to menu:Systems[Autoinstallation > Profiles] and selecting the profile you want to edit.
-Navigate to the menu:System Details[File Preservation] subtab and select the file preservation list to include.
+One of the possibilities offered by the {kickstart} profiles is the file preservation lists. If you have many custom configuration files located on a client you want to {kickstart}, you can save them as a list, and associate that list with the {kickstart} profile to be used.
 
+.Procedure: Create a File Preservation List
+. In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > File Preservation] and click btn:[Create File Preservation List].
+. Enter a suitable label, and list absolute paths to all files and directories you want to save.
+. Click btn:[Create List].
+. Include the file preservation list in your {kickstart} profile by navigating to menu:Systems[Autoinstallation > Profiles] and selecting the profile you want to edit. Navigate to the menu:System Details[File Preservation] subtab and select the file preservation list to include.
 
 [NOTE]
 ====
 File preservation lists are limited to a total size of 1{nbsp}MB.
-Special devices like [path]``/dev/hda1`` and [path]``/dev/sda1`` are not supported.
+Special devices like [path]``/dev/hda1`` and [path]``/dev/sda1`` cannot be preserved.
 Only use file and directory names, you cannot use regular expression wildcards.
 ====
+
+For more information about Kickstart, see the Red Hat documentation.
+
+
+[[templates-syntax]]
+== Templates Syntax
+
+Parts of your installation file will be replaced during the installation:
+
+* variables will be replaced with single values;
+* code snippets will be replaced with whole sections of text;
+* escaped symbols or sections will not be replaced.
```

Comment:
```suggestion
* Variables are replaced with single values.
* Code snippets are replaced with whole sections of text.
* Escaped symbols or sections are not replaced.
```

Not sure about including that third point, which probably means this should be prose instead.

Reply:
escaping  is a section of its own

but yeah it should not be part of that list